.github/workflows/release-pr-coderabbit-gate.yml

```
@@ -0,0 +1,42 @@
+  coderabbit-release-gate:
+    runs-on: ubuntu-latest
+    # Only run for release PRs /title check as initial filter
+    if: |
+      github.event.pull_request &&
+      (github.event.pull_request.author_association == 'MEMBER' ||
+      github.event.pull_request.author_association == 'OWNER') &&
+      startsWith(github.event.pull_request.title, 'chore: release v') ||
+      startsWith(github.event.pull_request.title, 'release v')
+
+    steps:
+      - name:  Harden the runner
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)